pulsar-client/src/main/java/org/apache/pulsar/client/impl/MessageImpl.java

```diff
@@ -139,11 +131,8 @@
         this.schema = schema;
     }
 
-    public MessageImpl(String msgId, Map<String, String> properties, byte[] payload, Schema<T> schema) {
```

Comment:
Why do we delete the one with `byte[]` ?    No one using it now?

Reply:
it was redundant. It was only used in one place which could easily create the ByteBuf from unpooled.

In general, it's best to keep the number of constructors low. There's already far too many for MessageImpl, but I don't want to tackle the whole thing in this patch.